tests/components/growatt_server/sensor/test_sensor_init.py

```diff
@@ -0,0 +1,121 @@
+"""Tests for the Growatt server sensor integration."""
+
+from unittest.mock import patch
+
+import pytest
+
+from homeassistant.components.growatt_server.const import (
+    DEFAULT_PLANT_ID,
+    LOGIN_INVALID_AUTH_CODE,
+    LOGIN_LOCKED_CODE,
+)
+from homeassistant.components.growatt_server.sensor import get_device_list
```

Comment:
We should not touch internals directly, we should rather patch the library and have HA set up the integration